scripts/bootstrap-JulietCSharp.sh

```diff
@@ -34,16 +34,18 @@ if [[ "$EXIT_ON_ERROR" = "true" ]]; then
   set -e
 fi
 
-cd $BASE_DIR
+cd "$BASE_DIR"
 
 if [ ! -d "JulietCSharp" ]; then
   echo "Downloading JulietCSharp..."
   curl -o JulietCSharp.zip "https://samate.nist.gov/SARD/downloads/test-suites/2020-08-01-juliet-test-suite-for-csharp-v1-3.zip"
   julietsha=$(sha256sum JulietCSharp.zip)
-  if [ $julietsha -ne "2e6dbac4741fb020a0b1c2db69e98aed165987df2bd70bd51f7c8c5302c8e8f8" ]; then
-    echo "sha256 failed for the Juliet download!"
-    rm JulietCSharp.zip
-    exit 2
+  if [[ "$UPDATE_BENCHMARKS" = "false" ]]; then
```

Comment:
I'm not sure if updating Juliet C# is a good idea. Current scripts rely on fixes relevant only for the old versions of .NET, and most likely will break if Juliet C# does get updated in the future.

Reply:
With this option, there will be at least a proper way to get the updated benchmark to fix the scripts. This `update` option serves only development purposes.